README.md

```diff
@@ -9,17 +9,22 @@
 
 # AtlasReader
 
-This package provides a Python interface for generating coordinate tables and region labels from statistical MRI images.
+This package provides a Python interface for generating coordinate tables and
+region labels from statistical MRI images.
 
 If you are using `atlasreader` in your publication, please cite the following paper:
 
-Markello R., Gale D., Herholz P., Whitaker K., & Notter M. P. (2019). AtlasReader: Quick generation of coordinate tables and region labels for statistical MRI images. *The Journal of Open Source Software*, ZZZ, YYY.
+Notter M. P., Gale D., Herholz P., Markello R. D., Notter-Bielser M.-L., & Whitaker K. (2019). AtlasReader: A Python package to generate coordinate tables, region labels, and informative figures from statistical MRI images. *The Journal of Open Source Software*, ZZZ, YYY.
```

Comment:
Can we have this in if it *hasn't* been accepted yet?? Why not archive on zenodo and add that doi? We can replace it with this citation once we've been accepted at JOSS.

I'm being a bit picky BUT if I were reviewing I'd probably be put off a little if I saw "cite as JOSS paper" in the README file 😬 Don't want to put reviewers in a bad mood too early!

Reply:
Oh, and I saw a thing about author order....I'll reply to that somewhere...or come back here if it's already resolved.

Reply:
Commented above: https://github.com/miykael/atlasreader/pull/66#discussion_r251519656